src/setup/scss/mixins/_font.scss

```diff
@@ -1,3 +1,7 @@
+@use './font-size';
```

Comment:
we need to @use the direct file, using `@use '../mixins'` leads to a import loop error